internal/pkg/remote/endpoint/service.go

```diff
@@ -191,17 +195,6 @@ func (config *Config) GetAllServices() (map[string][]Service, error) {
 // GetServiceURI returns the URI for the service at the specified endpoint
 // Examples of services: consent, library, key, token
 func (config *Config) GetServiceURI(service string) (string, error) {
-	// don't grab remote URI if the endpoint is the
-	// default cloud service
-	if config.URI == DefaultCloudURI {
-		switch service {
-		case Library:
-			return DefaultLibraryURI, nil
-		case Keyserver:
-			return DefaultKeyserverURI, nil
-		}
-	}
-
```

Comment:
I think it's a problem to remove these defaults as well.

Reply:
Reintegrated again.